.rubocop_todo.yml

```diff
@@ -3,9 +3,7 @@
 
 # Offense count: 19
 Metrics/AbcSize:
-  Exclude:
-    - 'test/dummy/db/migrate/20110208155312_set_up_test_tables.rb'
-  Max: 46 # Goal: 15
+  Max: 50
```

Comment:
I don't understand why the `Max` increased here. `Max` should either stay the same, or decrease.